src/client/components/segment-bubble/segment-bubble.tsx

```diff
@@ -81,13 +83,49 @@ export class SegmentBubble extends React.Component<SegmentBubbleProps, SegmentBu
     }
   }
 
+  getReplacedUrl(label: string) {
+    var { urls, datum } = this.props;
+    var idx = 0;
+    if (urls.length !== 1) {
+      const nestIndex = (datum['__nest'] as number) - 1;
+      idx = nestIndex;
+    }
+    var url = urls[idx];
+    return url.replace('%s', label);
+  }
+
+  getHasUrl() {
+    const { urls, datum } = this.props;
+    var index = 0;
+    if (hasOwnProperty(datum, '__nest')) index = datum['__nest'] as number - 1;
+    return Boolean(urls[index]);
+  }
+
+
   renderMoreMenu() {
     const { moreMenuOpenOn } = this.state;
     if (!moreMenuOpenOn) return null;
-
     var menuSize = Stage.fromSize(160, 160);
     var label = this.getLabel();
 
+    var bubbleListItems = [
+      <li
+        key="copyValue"
+        className="clipboard"
+        data-clipboard-text={label}
+        onClick={this.closeMoreMenu.bind(this)}
+      >{STRINGS.copyValue}</li>
+    ];
+    if (this.getHasUrl()) {
+      bubbleListItems = bubbleListItems.concat(
```

Comment:
this could just as well be push as well as concat since it is all contained in one place (there is no danger of mutation)